src/Microsoft.DotNet.XHarness.iOS.Shared/TestReporter.cs

```diff
@@ -580,7 +580,7 @@ public async Task<(TestExecutingResult ExecutingResult, string ResultMessage)> P
             {
                 result.ExecutingResult = TestExecutingResult.TimedOut;
             }
-            else if (crashed)
+            else if (crashed || launchFailure)
```

Comment:
Approving, but maybe changing from TestExecuting.Crashed to LauchFailure would make more sense.